tests/s3/test_s3_actions.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+from unittest.mock import MagicMock, patch
+
+import pytest
+from botocore.exceptions import ClientError
+from chaoslib.exceptions import FailedActivity
+
+from chaosaws.s3.actions import delete_object, toggle_versioning
+
+data_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
+
```

Comment:
@joshuaroot to be a pain, would you mind `typing` the functions in this file similarly to how you have in the actions/probes?

At some point this package will have `mypy` added to it for linting and the more pre-emptive typing we can get in, the better :D

Reply:
Sure, not a problem.